smart_card_connector_app/src/application_unittest.cc

```diff
@@ -840,6 +840,54 @@ TEST_F(SmartCardConnectorApplicationSingleClientTest,
             SCARD_STATE_CHANGED | SCARD_STATE_UNAVAILABLE));
 }
 
+// Test `SCardGetStatusChange()` call from JS returns the reader and card
+// information.
+TEST_F(SmartCardConnectorApplicationSingleClientTest,
+       SCardGetStatusChangeWithCard) {
+  // Arrange:
+  TestingSmartCardSimulation::Device device;
+  device.id = 123;
+  device.type = TestingSmartCardSimulation::DeviceType::kGemaltoPcTwinReader;
+  device.card_type = TestingSmartCardSimulation::CardType::kCosmoId70;
+  SetUsbDevices({device});
+  StartApplication();
+  SetUpJsClient();
+  SetUpSCardContext();
+
+  // Act:
+  std::vector<Value> reader_states;
+  // This call is expected to return immediately, since we pass
+  // `SCARD_STATE_UNKNOWN`.
+  EXPECT_EQ(SimulateGetStatusChangeCallFromJsClient(
+                kFakeHandlerId, scard_context(),
+                /*timeout=*/INFINITE,
+                ArrayValueBuilder()
+                    .Add(DictValueBuilder()
+                             .Add("reader_name", "Gemalto PC Twin Reader 00 00")
```

Comment:
nit: We are using this Gemalto string a lot, we can probably use a constant for it